frontend/packages/dev-console/src/images/pipeline.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><defs><style>.cls-1{fill:#000;}.cls-2{fill:#000;}.cls-3{fill:none;stroke:#fff;stroke-miterlimit:10;}.cls-4{fill:#fff;}.cls-5{fill:#fff;}</style></defs><title>Red_Hat-Openshift4-Catalog_Icons-Pipelines</title><circle class="cls-1" cx="50" cy="50" r="50"/><path class="cls-2" d="M85.36,14.64A50,50,0,0,1,14.64,85.36Z"/><circle class="cls-3" cx="50" cy="50" r="27.55"/><path class="cls-3" d="M33.81,27.71a27.56,27.56,0,0,1,34.64,1.82"/><polygon class="cls-4" points="64.66 31.49 72.29 33.81 70.48 26.04 64.66 31.49"/><path class="cls-3" d="M27.71,66.19a27.56,27.56,0,0,1,1.82-34.64"/><polygon class="cls-4" points="31.49 35.34 33.81 27.71 26.04 29.52 31.49 35.34"/><path class="cls-3" d="M66.19,72.29a27.56,27.56,0,0,1-34.64-1.82"/><polygon class="cls-4" points="35.34 68.51 27.71 66.19 29.52 73.96 35.34 68.51"/><path class="cls-3" d="M72.29,33.81a27.56,27.56,0,0,1-1.82,34.64"/><polygon class="cls-4" points="68.51 64.66 66.19 72.29 73.96 70.48 68.51 64.66"/><path class="cls-4" d="M44.86,27.93a7.28,7.28,0,1,1,10.31,0,7.16,7.16,0,0,1-10.31,0"/><path class="cls-5" d="M44.86,27.93,55.15,17.64a7.33,7.33,0,0,1,0,10.31,7.16,7.16,0,0,1-10.31,0"/><path class="cls-4" d="M44.86,82.66a7.28,7.28,0,1,1,10.31,0,7.17,7.17,0,0,1-10.31,0"/><path class="cls-5" d="M44.86,82.66,55.15,72.37a7.33,7.33,0,0,1,0,10.31,7.17,7.17,0,0,1-10.31,0"/><path class="cls-4" d="M17.31,55.11a7.27,7.27,0,1,1,10.3,0,7.16,7.16,0,0,1-10.3,0"/><path class="cls-5" d="M17.31,55.11,27.59,44.82a7.33,7.33,0,0,1,0,10.31,7.16,7.16,0,0,1-10.3,0"/><path class="cls-4" d="M72.41,55.11a7.27,7.27,0,1,1,10.3,0,7.16,7.16,0,0,1-10.3,0"/><path class="cls-5" d="M72.41,55.11,82.69,44.82a7.33,7.33,0,0,1,0,10.31,7.16,7.16,0,0,1-10.3,0"/></svg>
```

Comment:
This is a UX approved icon?

Reply:
We're working on getting another SVG, but it's very similar yes :) It's the b&w version of the operator logo